apidoc/Titanium/UI/Label.yml

```diff
@@ -157,10 +157,10 @@ properties:
     since: "5.4.0"
         
   - name: maxLines
-    summary: Makes the label at most this many lines tall.
-    platforms: [android]
+    summary: Makes the label at most this many lines tall. 
+    platforms: [android, iphone, ipad]
     type: Number
-    since: "4.1.0"
+    since: {android: "4.1.0", ios: "6.0.0"}
```

Comment:
Rename to  `{android: "4.1.0", iphone: "6.1.0", ipad: "6.1.0}`
- 6.0.0 is already and Beta and will not take new features at this point
- The docs need to distinguish `iphone` and `ipad` as specified keys